src/Files.App/Views/Layouts/GridLayoutPage.xaml.cs

```diff
@@ -183,27 +183,35 @@ override public void StartRenameItem()
 
 			int extensionLength = RenamingItem.FileExtension?.Length ?? 0;
 
-			GridViewItem gridViewItem = FileList.ContainerFromItem(RenamingItem) as GridViewItem;
-			if (gridViewItem is null)
+			if (FileList.ContainerFromItem(RenamingItem) is not GridViewItem gridViewItem)
 				return;
 
-			TextBox textBox = null;
+			if (gridViewItem.FindDescendant("ItemName") is not TextBlock textBlock)
+				return;
+
+			TextBox? textBox = null;
 
 			// Handle layout differences between tiles browser and photo album
-			if (FolderSettings.LayoutMode == FolderLayoutModes.GridView)
+			if (FolderSettings?.LayoutMode == FolderLayoutModes.GridView)
 			{
-				Popup popup = gridViewItem.FindDescendant("EditPopup") as Popup;
-				TextBlock textBlock = gridViewItem.FindDescendant("ItemName") as TextBlock;
+				if (gridViewItem.FindDescendant("EditPopup") is not Popup popup)
+					return;
+
 				textBox = popup.Child as TextBox;
+				if (textBox is null)
+					return;
+
 				textBox.Text = textBlock.Text;
 				textBlock.Opacity = 0;
 				popup.IsOpen = true;
 				OldItemName = textBlock.Text;
 			}
 			else
 			{
-				TextBlock textBlock = gridViewItem.FindDescendant("ItemName") as TextBlock;
 				textBox = gridViewItem.FindDescendant("TileViewTextBoxItemName") as TextBox;
+				if (textBox is null)
+					return;
```

Comment:
```suggestion
				if (gridViewItem.FindDescendant("TileViewTextBoxItemName") is not TextBox textBox)
					return;
```

Reply:
`textBox` is already defined on 192 so I don't think this will work